modules/ROOT/pages/tutorial/access-control.adoc

```diff
@@ -740,13 +1736,15 @@ MATCH (n:Patient)
 DETACH DELETE n;
 ----
 
-[role=erroronlyqueryresult]
+.Result
+[queryresult]
 ----
-org.neo4j.graphdb.ConstraintViolationException: Cannot delete node<42>, because it still has relationships. To delete this node, you must first delete its relationships.
+Cannot delete node<65>, because it still has relationships. To delete this node, you must first delete its relationships.
 ----
 
-The reason why this query fails is that, while Bob can find the `(:Patient)` node, they do not have sufficient traverse rights to find nor delete the outgoing relationships from it.
-Either they need to ask Tina the `itadmin` for help for this task, or more privileges can be added to the `receptionist` role:
+The reason why this query fails is that, while Bob can find the `(:Patient)` node, he does not have sufficient traverse rights to find nor delete the outgoing relationships from it.
```

Comment:
maybe change to something like this?

`he does not have sufficient traverse rights to find nor the rights to delete the outgoing relationships from it.`